config/vufind/searches.ini

```diff
@@ -628,6 +628,9 @@ view=full
 ; in the default recommendations section. To set the configuration settings 
 ; for this feature, adjust the parameters below.
 [MapSelection]
+; These configuration settings have been superseded by the geofeatures.ini file.
+; See the [MapSelection] section of the geofeatures.ini file for more information.
+;
```

Comment:
As noted above, we should delete redundant documentation here. Also note that the [MapSelection] section is referenced a couple other places earlier in this file. We should update those references too if necessary.

Reply:
Will do. Thanks for the note.